waspc/cli/Wasp/Cli/Command/Common.hs

```diff
@@ -57,3 +61,26 @@ alphaWarningMessage :: String
 alphaWarningMessage =
   "NOTE: Wasp is still in Alpha, therefore not yet production ready "
     ++ "and might change significantly in the future versions."
+
+toWaspMessage :: String -> String
+toWaspMessage = waspMessage ""
+
+toWaspStartMessage :: String -> String
+toWaspStartMessage = waspMessage "🐝"
+
+toWaspSuccessMessage :: String -> String
+toWaspSuccessMessage = waspMessage "✅"
+
+toWaspFailureMessage :: String -> String
+-- Add a bit more padding on errors for more pronounced
+-- visibility and better display of any following error context.
+toWaspFailureMessage str = "\n" ++ waspMessage "❌" str ++ "\n"
+
+waspMessage :: String -> String -> String
+waspMessage emoji message = unwords ["\n", prefix, message, suffix, "\n"]
+  where
+    prefix = emoji ++ " ---"
+    prefixAndMessageLength = length prefix + length message
+    idealLength = 80
```

Comment:
Not sold on `idealLength` either, but it is not a true `maxLength` since if `prefixAndMessageLength` is > 80 our message will be `prefixAndMessageLength + length suffix`, so that is sort of misleading.

Reply:
I like this, and I think naming is good! `idealLength` gave me a pretty clear idea of what is happening. You could name it `minLength`, but I don't think it is that clear what this means.